haskell/test/Test/Ligo/TreasuryDAO.hs

```diff
@@ -184,10 +183,10 @@ flushXtzTransfer = uncapsNettest $ withFrozenCallStack $ do
         }
 
   -- Advance one voting period to a voting stage.
-  advanceTime (sec 10)
+  advanceLevel 10
   withSender dodOwner2 $ call dodDao (Call @"Vote") [upvote]
   -- Advance one voting period to a proposing stage.
-  advanceTime (sec 10)
+  advanceLevel 10
```

Comment:
Since you are going through the trouble of updating all these, I wonder if we can make an `advanceStage` that can read the `period` length from the `DaoOriginateData` and move as many levels as necessary.

Reply:
It seems that voting period is not part of `DaoOriginateData`. Or your suggestion also include adding it in `DaoOriginateData`?

Reply:
Ah yeah, I didn't check that, but I think `DaoOriginateData` should indeed contain the `period` IMO.

In such case these lines could be changed to:
```suggestion
  advanceLevel dodPeriod
```
instead.

Reply:
Done.